pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -53,6 +53,7 @@ func (reconciler *Reconciler) Reconcile(request reconcile.Request) (reconcile.Re
 		} else {
 			log.Error("Can't properly create ConfigMaps. ", err)
 		}
+		deployConsole(reconciler, myDep)
```

Comment:
this should probably be moved up under line 52 (reconciler.CreateConfigMaps) ... need to be sure `myDep` was successfully pulled first.